### PR TITLE
Adding public religious channels from Guatemala

### PR DESCRIPTION
**Pull request by aalonzolu**
Departments added

- > Guatemala
- > Huehuetenango
- > Quetzaltenango

More channels at: https://oursnetworktv.com/#/Dashboard

### CODE REVIEW

File: streams/gt.m3u [hunk 1]
```diff
@@ -11,3 +11,23 @@ http://streamingcontrol.com:1935/ectv/ectv/playlist.m3u8
 https://devdbm9fvg6q0.cloudfront.net/chapintv/smil:canal7.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="TN23.gt" status="error",TN 23 (480p)
 https://devdbm9fvg6q0.cloudfront.net/chapintv/smil:canal23.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="BibliaCentral.gt" status="online" group-title="Religious",Bíblica-Central
+https://stream.oursnetworktv.com/latin/BibliaCentral/playlist.m3u8
+#EXTINF:-1 tvg-id="feViviente.gt" status="online" group-title="Religious",Fe Viviente
+https://stream.oursnetworktv.com/latin/feVivienteGtm/playlist.m3u8
+#EXTINF:-1 tvg-id="nissiGtmH" status="online" group-title="Religious",Jehová Nissi
```
Comment (LaneSh4d0w) [inline streams/gt.m3u:18]:
```suggestion
#EXTINF:-1 tvg-id="JehovaNissi.gt" status="online" group-title="Religious",Jehová Nissi
```

File: streams/gt.m3u [hunk 2]
```diff
@@ -11,3 +11,23 @@ http://streamingcontrol.com:1935/ectv/ectv/playlist.m3u8
...
+https://stream.oursnetworktv.com/latin/nissiGtmH/playlist.m3u8
+#EXTINF:-1 tvg-id="penielmusical.gt" status="online" group-title="Religious",Peniel Musical Un Encuentro Con Dios
+https://stream.oursnetworktv.com/latin/eC3pEmUsYKmcaLeJ/playlist.m3u8
+#EXTINF:-1 tvg-id="remaPresencia.gt" status="online" group-title="Religious",Presencia TV
+https://stream.oursnetworktv.com/latin/remaPresencia/playlist.m3u8
+#EXTINF:-1 tvg-id="radioTGBAgtm" status="online" group-title="Religious",Radio Maya TGBA
```
Comment (LaneSh4d0w) [inline streams/gt.m3u:24]:
```suggestion
#EXTINF:-1 tvg-id="RadioMayaTGBA.gt" status="online" group-title="Religious",Radio Maya TGBA
```

**Review by aalonzolu — COMMENTED** [review]
Thanks for the correction!

**Comment by LaneSh4d0w** [comment]
> Thanks for the correction!

Are you someone that manages this hosting service ? We've seen a few from oursnetwork, but not that many.

**Review by BellezaEmporium — DISMISSED** [review]

**Review by freearhey — DISMISSED** [review]

**Review by sguinetti — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**